book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    @rentals.push(Rental.new(date, person))
+  end
+end
```

Comment:
- [ ] Rental class constructor also expects `book` as a parameter so please make sure to pass the `book` parameter in the Rental class constructor. You can consider the below code.

```ruby
  def add_rental(date, person)
    @rentals.push(Rental.new(date,person,self))
  end